ring_buffer/ring_buffer.py

```diff
@@ -1,7 +1,7 @@
 class RingBuffer:
     def __init__(self, capacity):
         self.capacity = capacity
-        self.list = ['bla'] * capacity
+        self.list = [None] * capacity
         self.capacity_counter = 0
     def append(self, item):
         if self.capacity_counter < self.capacity:
```

Comment:
try to recount and rethink this again
let's say if we start at position 0 in the list, and the capacity is 3, what is the size of the array and what is the position of the end? ;)